tools/src/test/python/dlpx/virtualization/_internal/test_cli.py

```diff
@@ -471,11 +471,11 @@ def test_file_not_exist(artifact_file):
 
         assert result.exit_code == 2
         assert result.output == (u'Usage: delphix-sdk upload [OPTIONS]'
-                                 u'\nTry "delphix-sdk upload -h" for help.'
+                                 u'\nTry \'delphix-sdk upload -h\' for help.'
                                  u'\n'
-                                 u'\nError: Invalid value for "-a" /'
-                                 u' "--upload-artifact": File'
-                                 u' "/not/a/real/file/artifact.json"'
+                                 u'\nError: Invalid value for \'-a\' /'
+                                 u' \'--upload-artifact\': File'
+                                 u' \'/not/a/real/file/artifact.json\''
                                  u' does not exist.'
                                  u'\n')
```

Comment:
same here and elsewhere

Reply:
That's what I was going to do but then I would have to change the entire test_cli.py to be consistent. I will have to check what convention we followed in other files.

Reply:
I just read the following recommendation from PEP8 (https://www.python.org/dev/peps/pep-0008/):

> In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a recommendation for this. Pick a rule and stick to it. When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability.

I will get rid of the backslashes.